chart/values.yaml

```diff
@@ -434,3 +434,23 @@ postgresql:
   enabled: true
   postgresqlPassword: postgres
   postgresqlUsername: postgres
+# Git sync
+gitSync:
+  enabled: true
+  # ssh example ssh://git@github.com/aneesh-joseph/dags-repo.git
+  # ssh example: git@github.com:aneesh-joseph/dags-repo.git
+  # https example: https://github.com/aneesh-joseph/dags-repo.git
+  #TODO Change the URLs used for testing
+  repo: https://github.com/aneesh-joseph/dags-repo.git
```

Comment:
Please change this to the apache project.

Reply:
done updated it to use dags from https://github.com/apache/airflow/tree/master/dags